robocopkaka/bootcamp-29-project
# #158356139 Fix api documentation

#### What does this PR do?
fix existing api documentation
#### Description of Task to be completed?
* create new `swagger.json` file inside a `docs` folder in the `server` directory
* move all existing documentation to this file
* add more documentation
* ensure that my endpoints can be tested from a browser
#### How should this be manually tested?
* clone this repo
* `cd` into the project folder
* run `npm install` to install all dependecies
* run `npm install -g sequelize-cli` to install `sequelize` globally
* run `npm run start:all` to start up the application
* go to `http://localhost:8000/docs` on your browser
* select any of the endpoints and click `Try it out` to ensure that it works
* make sure that you first use any of the `Users` tagged endpoint to either log in or signup to get a token before testing a protected endpoint
#### Any background context you want to provide?
users couldn't test my endpoints on a browser prior to this PR being raised
#### What are the relevant pivotal tracker stories?(if applicable)
#158356139
